neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -131,9 +131,11 @@ def proxies(self):
     @with_api_exceptions_handler
     def get_project(self, project_qualified_name):
         try:
-            project = self.backend_swagger_client.api.getProject(
-                projectIdentifier=project_qualified_name
-            ).response().result
+            response = self.backend_swagger_client.api.getProject(projectIdentifier=project_qualified_name).response()
+            warning = response.metadata.headers.get('X-Server-Warning')
```

Comment:
Can we use API that is capable of handling multiple `X-Server-Warning` response headers?